packages/manager/modules/pci/src/components/project/storages/databases/plan.class.js

```diff
@@ -0,0 +1,105 @@
+import find from 'lodash/find';
+import groupBy from 'lodash/groupBy';
+import maxBy from 'lodash/maxBy';
+import minBy from 'lodash/minBy';
+import reduce from 'lodash/reduce';
+import some from 'lodash/some';
+
+import Region from './region.class';
+
+export default class Plan {
+  constructor({ name, description }, availability, flavors) {
+    Object.assign(this, {
+      availability,
+      description,
+      name,
+    });
+    this.regions = reduce(
+      groupBy(this.availability, 'region'),
+      (regions, regionPlans, regionName) => [
+        ...regions,
+        new Region(
+          { name: regionName },
+          regionPlans,
+          flavors.filter((flavor) =>
+            find(regionPlans, ['flavor.name', flavor.name]),
+          ),
+        ),
+      ],
+      [],
+    );
+    this.isDefault = some(availability, 'default');
+  }
+
+  get minHourlyPrice() {
+    const { hourlyPrice } = minBy(
+      this.availability,
+      'hourlyPrice.priceInUcents',
+    );
+    return {
+      priceInUcents: hourlyPrice.priceInUcents * this.minNodes,
+      tax: hourlyPrice.tax * this.minNodes,
+    };
+  }
+
+  get minMonthlyPrice() {
+    const { monthlyPrice } = minBy(
+      this.availability,
+      'monthlyPrice.priceInUcents',
+    );
+    return {
+      priceInUcents: monthlyPrice.priceInUcents * this.minNodes,
+      tax: monthlyPrice.tax * this.minNodes,
+    };
+  }
+
+  get minCores() {
+    return minBy(this.availability, (currentPlan) => currentPlan.flavor.core)
```

Comment:
wont `minBy(this.availability, 'currentPlan.flavor.core')` do the same thing ?

Reply:
Yeah... This has been changed. Good one!